src/node_url.cc

```diff
@@ -1598,9 +1598,12 @@ void URL::Parse(const char* input,
                    special_back_slash) {
           if (buffer.size() > 0) {
             int port = 0;
```

Comment:
I think it might even get a little clearer now if you make this `unsigned`

Reply:
Yes, makes sense. Changed.